nix/overlay.nix

```diff
@@ -0,0 +1,25 @@
+# Override some packages and utilities in 'pkgs'
+# and make them available globally via callPackage.
+#
+# For more details see:
+# - https://nixos.wiki/wiki/Overlays
+# - https://nixos.org/nixos/nix-pills/callpackage-design-pattern.html
+self: super:
+let
+  inherit (super);  
```

Comment:
Completely unnecessary. And please use `final` and `prev`. With `final`/`prev` it's kinda obvious that `prev` is the version of `nixpkgs` before the overlay was applied and `final` is the version after the overlay was applied.